ppr-ui/package.json

```diff
@@ -10,7 +10,7 @@
     "preview": "vite preview --port 8080",
     "lint": "eslint . --ext .js,.ts,.vue",
     "lint:fix": "eslint . --ext .js,.ts,.vue --fix",
-    "test:unit": "vitest run",
+    "test:unit": "NODE_OPTIONS=--max-old-space-size=8192 vitest run",
```

Comment:
Is this the min value where tests are passing? Just curious where this number came from...

Reply:
This is just double the default amount, we may not need this by limiting the threads but wanted to be sure. 
This is equivalent to 8gigs (ie 8192 megabytes). 

-old-space is part of where the long-lived objects are stored, seemed like the right thing to increase due to the error we were bumping into. 

We should try and remove this if limiting the threads alone does the trick.